app/src/main/res/values/styles.xml

```diff
@@ -156,6 +156,12 @@
         <item name="behavior_expandedOffset">0dp</item>
     </style>
 
+    <style name="ExternalFileBottomSheetStyle" parent="BottomSheetStyle">
+        <item name="shapeAppearance">?attr/shapeAppearanceLargeComponent</item>
+        <item name="behavior_peekHeight">120dp</item>
+        <item name="behavior_fitToContents">true</item>
+    </style>
+
```

Comment:
Why do you need a different style than the style used for the `FileInfoActionsView`?

Reply:
I'll use the existing style but I have to have the fitToContents to true. I'll update the method which initialize the bottomSheetBehavior.

Reply:
Add a TODO to remove this variable when the bug that requires it is fixed and create a card for this bug